SparkPost/matchbox
# UX-565 Massively change ActionList and Popover interactions

### What Changed
- Changes the behavior of Popovers
  - Popovers are now rendered in a `Portal`
  - Popovers now correctly focus on a focusable child when opening
  - Popovers now open when hitting the up or down arrows when uncontrolled
  - Popovers now close when pressing tab (via a new prop)
- ActionList
  - ActionLists are now keyboard navigable with the up and down arrows
  - ActionLists now properly set `role="menuitem"` and `role="menu"`
- Table
  - Tables now only render one table with the `freezeFirstColumn` prop

### How To Test or Verify

- Run libby
- Visit http://localhost:9001/?path=Popover__with-an-ActionList
  - Verify the changes above
- On: http://localhost:9001/?path=Popover__uncontrolled
  - Verify the popover container is focused when opened
- On: http://localhost:9001/?path=Popover__controlled
  - Verify the close button is focused when opened
- On: http://localhost:9001/?path=Table__frozen-first-column
  - Verify popovers work correctly in the first column of the table

### PR Checklist
